db/options.go

```diff
@@ -16,11 +16,11 @@ const (
 	defaultDatastorePath = "eventstore"
 )
 
-// Option takes a Config and modifies it.
-type Option func(*Config) error
+// NewDBOption takes a Config and modifies it.
+type NewDBOption func(*NewDBOptions) error
```

Comment:
Needed to rename the options for creating DBs so decided to make them consistent with new option naming conventions.

Reply:
Sweeeeet